website/src/docs/hotchocolate/fetching-data/index.md

```diff
@@ -1,289 +1,49 @@
 ---
-title: "Resolver"
+title: Overview
 ---
 
-import { ExampleTabs } from "../../../components/mdx/example-tabs"
+In this section we will learn everything about fetching data with Hot Chocolate.
 
-Here we will learn what resolvers are, how they are defined, and what else we could do with them in Hot Chocolate.
+# Resolvers
 
-# Introduction
+Resolvers are the main building blocks when it comes to fetching data. Every field in our GraphQL schema is backed by such a resolver function, responsible for fetching the data associated with this field. Since a resolver is just a function we can use it to retrieve data from a database, a REST service or any other data source.
```

Comment:
```suggestion
Resolvers are the main building blocks when it comes to fetching data. Every field in our GraphQL schema is backed by such a resolver function, responsible for returning the field's value. Since a resolver is just a function, we can use it to retrieve data from a database, a REST service, or any other data source as needed. 
```